### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by PurpurTentakel97**

### CODE REVIEW

File: src/routes/impressum/+page.svelte [hunk 1]
```diff
@@ -0,0 +1 @@
+<h1>Bestes Impressum. Also wirklich.</h1>
```
Comment (mgerhold) [inline src/routes/impressum/+page.svelte:1]: missing newline at end of file (you can set VSCode to always end files with a newline upon save)

File: static/Style.css [hunk 2]
```diff
@@ -1,27 +1,34 @@
...
+        --third-color: #674c8a;
+        --text-color: white;
+    }
+}
 
 body {
     background-color: var(--background-color);
     transition: background-color var(--transition-duration);
-}
+}
```
Comment (mgerhold) [inline static/Style.css:34]: missing newline

**Review by mgerhold — CHANGES_REQUESTED** [review]
All in all I think the footer looks good, but I think the font should be smaller and the margins (or paddings?) can be smaller.
image
So I think all texts in the footer (and the logo) should be smaller, but the copyright notice should be even *more* small than the rest. What do you think?

File: src/elements/footer.svelte [hunk 3]
```diff
@@ -1,9 +1,97 @@
...
+		<nav>
+			<ul>
+				<li><a href="/login">Login</a></li>
+				<li><a href="/login">Register</a></li>
+				<li><a href="/impressum">Impressum</a></li>
+			</ul>
+		</nav>
+		
+		<nav>
+			<p class="YearHeadline">Alle Test-Conf:</p>
```
[the remark this answers is not in the document]
Comment (PurpurTentakel97) [reply src/elements/footer.svelte:12]: done

File: static/Style.css (hunk 2 above)
Comment (PurpurTentakel97) [reply static/Style.css:34]: The formatter does not add a newline here.
Diese nicht gut. 😅

Comment (mgerhold) [reply static/Style.css]: Correct, diese schlecht 😂

**Review by mgerhold — APPROVED** [review]
Nice work! 👍

File: src/elements/footer.svelte [hunk 4]
```diff
@@ -1,9 +1,98 @@
...
+		<p>
+			Bester Beschreibungstext <br />
+			aös,jkdhfga <br />
+			sgasdg asdgasdgasd <br />
+			gasdgökajsdhga <br />
+			sdgaölkjskdhg
+		</p>
 
-    <p>copyright 2024</p>
+		<img src={"/logo_big.png"} alt="The big Test-Conf logo. It shows the name on black clouds. Black lines emerge from the clouds with yellow dots at the ends. They look like circuits."/>
```
Comment (peddermaster2) [inline src/elements/footer.svelte:27]: In einer deutschen Seite, sollten auch die alt Texte in deutsch sein.

Comment (PurpurTentakel97) [reply src/elements/footer.svelte:27]: see #8 
I will close this.

**Pull request merged**